### ISSUE TITLE
Create 'install_sklearn.pp' manifest file

### ISSUE THREAD

**Issue by jeff1evesque** [jeff1evesque/machine-learning#1965]
After #1972 has been resolved, we will create `install_sklearn.pp`.  This file will be responsible for installing scikit-learn, and building from the github repository.

**Note:** we will need to include `installer.pp` into our `Vagrantfile`.
**Note:** we will need to delete the current versioned `scikit-learn` submodule.

**Comment by jeff1evesque** [comment]
Since we have created the `/puppet/modules/` directory via the vagrant-triggers vagrant plugin, the following within `install_sklearn.pp` is no longer needed:

```puppet
...
## create '/vagrant/build/' directory
file {'/vagrant/build/':
    ensure => 'directory',
    before => Vcsrepo['/vagrant/build/scikit-learn'],
}
...
```

**Comment by jeff1evesque** [comment]
We will reinsert the logic that was [removed](https://github.com/jeff1evesque/machine-learning/issues/1965#issuecomment-115696621).

**Comment by jeff1evesque** [comment]
We forgot to install scikit-learn after building it.

**Comment by jeff1evesque** [comment]
We will make a minor comment change.
